### DISCUSSION TITLE
Break end with no sound on macOS locked screen

### DISCUSSION THREAD

**Discussion by kristiannotari** [tom-james-watson/breaktimer-app#222] [Q&A] [answered]
Hello,

great app btw! I'm using to keep track of breaks I need to do while working. One thing I noticed is that when the break start, I usually lock my screen and get up. After locking the screen, when the break actually start (after the initial countdown) a sound is played warning me about the break start.

The same sound doesn't play when the break end, having the screen locked. That's unfortunate. Is there a way to have it play the sound when the X minutes break ends? It would be easier so to know when to come back to my desk :)

Thanks!

**Comment by tom-james-watson** [comment]
Breaks do play a sound when they end. If you're not hearing it it's probably something to do with locking the screen. Try doing a break without locking the screen and you should get the end break noise. I don't think there's anything we can do here to be honest.

**Reply by kristiannotari** [reply]
Ok so basically on macOS we can't get a break ending sound when the computer's locked? Like that being a macOS limitation or something not implemented in the app yet?

**Reply by torpesco** [reply]
I'm on Ventura 13.3.1 on an M1 Pro and get the end of break chime even when I've locked my computer.

**Reply by kristiannotari** [reply]
Don't know if related to https://github.com/tom-james-watson/breaktimer-app/issues/206 but I have the same OS version and M1 Pro too but it's not playing any sound when locked. I do have it shell closed and plugged to a docking station delivering power, so maybe that's causing that. If I unlock it some seconds before the sound should play, it correctly plays.

**Reply by torpesco** [reply]
Strange issue. I've had #206 a few times, too. For this one, though, I always get the chime, even if the lid's closed and I'm connected to a "dock". (Dell monitor supplying power.)

**Reply by kristiannotari** [reply, marked as answer]
Ok I got it working by checking the screens -> advanced -> "...prevent Mac from going to sleep when screen off..." option, which somehow wasn't checked.

You can read more about such option here:
https://osxdaily.com/2023/01/02/how-prevent-mac-sleeping-when-display-is-off/

**Reply by torpesco** [reply]
Oh, interesting. I just hadn't hit it because my breaks aren't long enough for my Mac to go to sleep in that time.
